### PR TITLE
Codes on turtle graphics

### LINKED ISSUE #744: Codes On Turtle Graphics
Design several figures using Turtle Graphics in Python.

I am a participant at GSSOC'21, requesting admins to assign me this issue.

### PR DESCRIPTION
**Pull request by ricsin23**
# Description

I have written two codes on Turtle Graphics which use the turtle Library of Python. This will solve issue  #744. No dependencies are required for this change.

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x]  Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update
- [ ] Documentation Update

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

File: Codes on Turtle Graphics/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Codes on Turtle Graphics
+
+_turtle_ is a pre installed library in Python that enables us to create pictures and shapes by providing them with a virtual canvas. The onscreen pen that we use to draw
+is called turtle.
+
+### Requirements
```
Comment (kaustubhgupta) [inline Codes on Turtle Graphics/README.md:6]:
```suggestion
## Requirements
```

File: Codes on Turtle Graphics/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+#### 1. Python Environment:
```
Comment (kaustubhgupta) [inline Codes on Turtle Graphics/README.md:7]:
```suggestion
### 1. Python Environment:
```

File: Codes on Turtle Graphics/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+Use applications like Python IDLE or Jupyter Notebook.
+#### 2. Python Version:
```
Comment (kaustubhgupta) [inline Codes on Turtle Graphics/README.md:9]:
```suggestion
### 2. Python Version:
```

File: Codes on Turtle Graphics/README.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+Ensure you have version 3 of Python.
+
+With the turtle library of Python, we can draw several shapes and figures. Here's two samples we can create with turtles's library.
+
+### 1. Animated Robot
```
Comment (kaustubhgupta) [inline Codes on Turtle Graphics/README.md:14]:
```suggestion
## 1. Animated Robot
```

File: Codes on Turtle Graphics/README.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+This is a simple figure we can create to understand basics of turtle graphics.  
+<img src="Robot.png" width="350" height="250">
+
+### 2. Kaleido Spiral
```
Comment (kaustubhgupta) [inline Codes on Turtle Graphics/README.md:18]:
```suggestion
## 2. Kaleido Spiral
```

File: Codes on Turtle Graphics/README.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+In this, we will draw circles and squares in continuation. Everytime we will change the radius of circle and side dimension of square to make it look like a expanding squares and circles.  
+<img src="Kaleido-spiral.png" width="350" height="250">
```
Comment (kaustubhgupta) [inline Codes on Turtle Graphics/README.md:20]: Images not rendered: 

image

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
The code is working, just some readme changes:

**Comment by kaustubhgupta** [comment]
@antrikshmisri again, turtle is not supported

**Review by kaustubhgupta — APPROVED** [review]

**Review by Kushal997-das — APPROVED** [review]
LGTM 💯

**Comment by antrikshmisri** [comment]
> @antrikshmisri again, turtle is not supported

Yup, did you get any response from the PA?

**Review by antrikshmisri — APPROVED** [review]

**Review by santushtisharma10 — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]
LGTM..

**Pull request merged**